src/Shared/Recruit.Shared.Web/ViewModels/InfoMessages.cs

```diff
@@ -6,6 +6,8 @@ public class InfoMessages
     {
         public const string VacancyUpdated = "The vacancy dates for “{0}” have been updated.";
 
+        public const string VacancyDeleted = "Vacancy VAC{0} - '{1}' has been deleted.";
+
         public const string VacancyClosed = "Vacancy VAC{0} - '{1}' has been closed.";
 
         public const string VacancyCloned = "Advert succesfully cloned";
```

Comment:
by the looks of things we will need some additional messages here for `AdvertUpdated`, `AdvertDeleted` etc. Also the `VacancyClosed` uses "Advert" so should be renamed.